markdown/extensions/attr_list.py

```diff
@@ -55,12 +65,12 @@ def _handle_word(s, t):
 ])
 
 
-def get_attrs(str):
+def get_attrs(str: str) -> list[Tuple[str, str]]:
```

Comment:
Future annotations let you use the `tuple` built-in directly, without having to import `Tuple` from `typing`.